cid/helpers/quicksight/dashboard.py

```diff
@@ -40,6 +40,18 @@ def deployedTemplate(self) -> CidQsTemplate:
     def deployedTemplate(self, template: CidQsTemplate) -> None:
         self._deployedTemplate = template
 
+    @property
+    def template_id(self) -> str:
+        if isinstance(self.deployedTemplate, CidQsTemplate):
+            return self.deployedTemplate.arn
```

Comment:
Shoudn't we use `self.deployedTemplate.id` here?

Reply:
indeed